### ISSUE TITLE
simulation 

### ISSUE THREAD

**Issue by xuquanxing** [yiyunevin/Globla-Planner#1]
Can the simulation robot use both Turtlebot3 and KUKAyoubot?

**Comment by yiyunevin** [comment]
Yes, you can modify the parameter "robot_radius" in .../config/base_global_planner_params.yaml depend on the size of the robot.

However, what the Global Planner focus on is to plan a trajectory before the robot start to move. The result is the position of the nodes.

If you are talking about getting the robot to the goal point, the project is done in the repository [Local-Planner](https://github.com/yiyunevin/RL-RRT-Local-Planner). It'll generate the moving command depend on the result of the Global Planner. And it can be applied on both Turtlebot3 and youBot.
